pkg/sql/executor_ir.go

```diff
@@ -243,6 +244,10 @@ func runSQLProgram(wr *PipeWriter, sqlProgram string, db *database.DB, modelDir
 		if e := runSingleSQLIR(wr, r, db, modelDir, cwd, session); e != nil {
 			return e
```

Comment:
It seems that we have to manually remove the temp dir in every return site in the loop 😓